test/config/chaosduck.yaml

```diff
@@ -24,6 +24,9 @@ metadata:
   name: chaosduck
   namespace: kourier-system
   labels:
+    app.kubernetes.io/name: chaosduck
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/part-of: knative-serving
```

Comment:
Any reason why you added it to the `test/config` resources? We don't release these.

Reply:
Only because they showed up in my rgrep... will remove it